src/workflows-app/WorkflowsInWorkspace.test.ts

```diff
@@ -106,4 +111,117 @@ describe('Workflows in workspace', () => {
       namespace: 'test-azure-ws-namespace',
     });
   });
+
+  it('should open workflow deletion confirmation modal when delete button is clicked', async () => {
+    const user = userEvent.setup();
+    const getWithVersions = jest.fn().mockReturnValue(Promise.resolve(methodDataWithVersions));
+    const getInfo = jest.fn().mockReturnValue(
+      Promise.resolve({
+        git: {},
+        build: {
```

Comment:
Do we want to have a test where this is not present, to prove the gating logic?

Reply:
good idea